backend/controllers/userController.js

```diff
@@ -1,14 +1,54 @@
-// userController.js
```

Comment:
I just wanted to do checks here for returning correct status codes, let me know if I should move them to the model